src/app/User.test.ts

```diff
@@ -58,20 +60,21 @@ describe(UserService.name, () => {
     it("Should pass the right data to db", async () => {
       // Arrange
       const date = new Date("Date Mon Oct 24 2022 17:21:08 GMT-0300");
-      const expectedValue = "2022-10-24T20:21:08.000Z";
       const { dbClientDouble, TableName, sut } = createTestContext();
 
       // Act
       await sut.registerReportsSectionVisit("email1", date);
 
       // Assert
       expect(dbClientDouble.update).toHaveBeenCalledWith({
-        TableName,
-        Key: { email: "email1" },
+        TableName: TableName,
+        Item: {
+          email: "email1",
+        },
         UpdateExpression:
           "set reportsSectionLastVisit = :reportsSectionLastVisit",
         ExpressionAttributeValues: {
-          ":reportsSectionLastVisit": expectedValue,
+          ":reportsSectionLastVisit": date.toISOString(),
```

Comment:
Oh, I prefer the original implementation, because it was explicit about what was the value format (exactly `2022-10-24T20:21:08.000Z`), without the need to go to [toISOString documentation](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)

Reply:
👍